addons/sale/models/sale_order.py

```diff
@@ -1361,56 +1392,64 @@ def _create_upsell_activity(self):
                 note=_("Upsell %(order)s for customer %(customer)s", order=order_ref, customer=customer_ref))
 
     def _prepare_analytic_account_data(self, prefix=None):
-        """
-        Prepare method for analytic account data
+        """ Prepare SO analytic account creation values.
 
-        :param prefix: The prefix of the to-be-created analytic account name
-        :type prefix: string
-        :return: dictionary of value for new analytic account creation
+        :param str prefix: The prefix of the to-be-created analytic account name
+        :return: `account.analytic.account` creation values
+        :rtype: dict
         """
+        self.ensure_one()
```

Comment:
Don't we `Note: self.ensure_one()` like in `payment`?

Reply:
not a fan of specifying it in the docstring personally :D